pkg/replicate/scheme.go

```diff
@@ -83,20 +86,7 @@ func (bf *BlockFilter) Filter(b *metadata.Meta) bool {
 
 	labelMatch := bf.labelSelector.Matches(blockLabels)
 	if !labelMatch {
-		selStr := "{"
-
-		for i, m := range bf.labelSelector {
-			if i != 0 {
-				selStr += ","
-			}
-
-			selStr += m.String()
-		}
-
-		selStr += "}"
-
-		level.Debug(bf.logger).Log("msg", "filtering block", "reason", "labels don't match", "block_labels", blockLabels.String(), "selector", selStr)
-
+		level.Debug(bf.logger).Log("msg", "filtering block", "reason", "labels don't match", "block_labels", blockLabels.String(), "selector", bf.labelSelectorStr)
```

Comment:
Adding  A comment ,where it describes that if the labelMatch value is 'False' we return False can be helpful

Also , can't we return the _labelMatch_ itself?? :thinking:

Reply:
I think the code you quoted says"labels don't match".

It's not readable to return the labelMatch directly because the matcher type is an enum, not a string.